packages/frontend/src/creation-form/payload-form/tvl.tsx

```diff
@@ -35,8 +35,11 @@ export const TvlPayloadForm = ({ t, payload, onChange }: PayloadFormProps) => {
                         inputWrapper: "w-full",
                     }}
                     label={t("label.tvl.protocol")}
-                    placeholder={t("placeholder.pick")}
+                    placeholder={t("placeholder.pick.protocol")}
```

Comment:
We should add `search` here to the `Select` to enable protocol filtering.

Reply:
Oh you're right. Addressed in 70e72afe084bf8fba301581095ecbb8f4a85c0f3